aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,177 @@
+==================  ==========  ============  ============  
+
+The bytes sent had a significant reduction with bundling as browsers are fairly verbose with the HTTP headers that they apply on requests. The load time shows a big improvement, however this example was run locally. You will get greater gains in performance when using bundling and minification with assets transferred over a network. To simulate the transfer time of a slow network, you can use the `Fiddler <http://www.telerik.com/fiddler>`_ tool. Within Fiddler, select the **Fiddler Rules** menu, **Performance** > **Simulate Modem Speeds**.
+
+Controlling Bundling and Minification
+-------------------------------------
+
+When working with an app created using the **ASP.NET 5 Web Application** template in Visual Studio, bundling and minification is enabled by setting the ``Hosting:Environment`` value to ``Production``. 
+
+The following steps allow you to set the ``Hosting:Environment`` value:
+
+1. In **Solution Explorer** of Visual Studio, right-click the project name and select **Properties**. 
+The property sheet for the app will be displayed.
+
+2. Set the value of the ``Hosting:Environment`` variable to ``Production`` and save your changes. 
+
+.. image:: bundling-and-minification/_static/04-AppPropertySheet.png
+
+The next time you run your app, the bundled and minified assets will be used rather than the regular assets. The **Web Application** template includes code that recognizes when it use the bundled and minified assets. 
+
+To see an example of the template uses the ``Hosting:Environment`` value, use the following steps:
+
+1. In **Solution Explorer**, find and open *Views/Shared/_Layout.cshtml*.
+
+2. Review the following HTML to see the different ``<environment>`` options:
+
+.. code-block:: html
+
+	<environment names="Development">
+		<link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+		<link rel="stylesheet" href="~/css/site.css" />
+	</environment>
+	<environment names="Staging,Production">
+		<link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.5/css/bootstrap.min.css"
+				asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
+				asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
+		<link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+	</environment>
+
+The *_Layout.cshtml* file uses the **environment** tag helper to provide two different versions of CSS files. One version of CSS files is for development and the other version is optimized for both staging and production. In Visual Studio 2015, when you change the **Hosting:Environment** environment variable to ``Production`` or ``Staging``, Visual Studio will build the Web app and link to the minimized CDN version. If the CDN request to the minimized file fails, the fallback version will be used.
+
+Using the "*" Wildcard Character to Select Files
```

Comment:
Is this text from the old Web Optimization stuff? I don't think we have any of these methods in ASP.NEt Core, do we?